### PR TITLE
[T] (quests) undergarden_enriching_water

### PR DESCRIPTION
**Pull request by AnzhiZhang**
Co-authored-by: Cactusstudent <97865546+Cactusstudent@users.noreply.github.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/undergarden_enriching_water.snbt [hunk 1]
```diff
@@ -241,7 +241,7 @@
 		{
 			x: -1.0d
 			y: 1.0d
-			description: ["9 shards make a bigger shard"]
+			description: ["9个utherium crystal碎片可以合成一个更大的utherium crystal碎片"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/undergarden_enriching_water.snbt]: 等模组翻译

Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/undergarden_enriching_water.snbt]:
```suggestion
			description: ["9个御腐破片可以合成一个更大的御腐水晶"]
```

File: config/ftbquests/quests/chapters/undergarden_enriching_water.snbt [hunk 2]
```diff
@@ -143,8 +143,8 @@
 			x: -1.0d
 			y: 0.0d
 			description: [
-				"Can be obtained from blank nuggets in undergarden enriching water."
-				"See JEI for chances"
+				"可以通过将空白粒丢入深园富集水的办法获得utheric shard。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/undergarden_enriching_water.snbt]:
```suggestion
				"可以通过将空白粒丢入深园富集水的办法获得御腐破片。"
```

File: config/ftbquests/quests/chapters/undergarden_enriching_water.snbt [hunk 3]
```diff
@@ -83,8 +83,8 @@
 			x: -5.0d
 			y: -1.0d
 			description: [
-				"Can be obtained from blank nuggets in undergarden enriching water."
-				"See JEI for chances"
+				"可以通过将空白粒丢入深园富集水的办法获得regalium crystal。Can be obtained from blank nuggets in undergarden enriching water."
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/undergarden_enriching_water.snbt]:
```suggestion
				"可以通过将空白粒丢入深园富集水的办法获得贵豪水晶。"
```

**Review by Cactusstudent — APPROVED** [review]
补翻译

**Pull request merged**